jerry-core/lit/lit-magic-strings.ini

```diff
@@ -170,6 +170,7 @@ LIT_MAGIC_STRING_ARGUMENTS_UL = "Arguments"
 LIT_MAGIC_STRING_EVAL_ERROR_UL = "EvalError"
 LIT_MAGIC_STRING_INT8_ARRAY_UL = "Int8Array"
 LIT_MAGIC_STRING_MAX_VALUE_U = "MAX_VALUE"
+LIT_MAGIC_STRING_MAX_SAFE_INTEGER_U = "MAX_SAFE_INTEGER"
```

Comment:
Wrong place of the new magic string. Magic strings are sorted by size at first. Please also check the comment at the beginning of the file. Did you run `tools/gen-magic-strings.py`?
```c
# List of ECMA magic strings
 #
 # These strings must be ascii strings. The NULL character cannot be part of
 # magic strings, because it must be the terminator character of all magic
 # strings.
 #
 # If the list is modified, tools/gen-magic-strings.py must be executed!
```